10.detect-separation/scripts/nbconverted/1.separate.py

```diff
@@ -0,0 +1,324 @@
+
+# coding: utf-8
+
+# # Detect Separation Between Two Phenotypes in TCGA, GTEx, and TARGET Data
+# 
+# **Gregory Way, 2019**
+# 
+# Perform a t-test to isolate the specific k dimension and algorithm that best distinguishes the two groups.
```

Comment:
This wording is a bit confusing

Reply:
Ah, thanks for pointing out - will address in next commit!